File: java/rest-simple-exercise/src/main/java/com/example/restsimple/model/Student.java
package com.example.restsimple.model;

import javax.persistence.*;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;
import java.time.LocalDateTime;

@Entity
@Table(name = "student")
public class Student {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "mnr", nullable = false)
    private String mnr;

    public Student() {

    }
}

File: kotlin/rest-simple-exercise/src/main/kotlin/com/example/restsimple/model/Student.kt
package com.example.restsimple.model

import java.time.LocalDateTime
import javax.persistence.*
import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

@Entity
@Table(name = "student")
class Student {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "mnr", nullable = false)
    var mnr: String? = null

    constructor()
}
